src/cmd/services/m3coordinator/model/rollup.go

```diff
@@ -0,0 +1,9 @@
+package model
```

Comment:
this should live somewhere more common. maybe `src/metrics` ?

Reply:
Looks like src/metrics has its own default model so I think keeping this "override" set in m3coordinator makes sense for now.